### ISSUE TITLE
Support Markdown tables as an output format

### ISSUE THREAD

**Issue by Fluxie** [google/benchmark#441]
Markdown support helps to streamline the publishing of the results in GitHub.  I.e. remove the need to reformat the console output to Markdown table format when publishing the results in GitHub.

**Comment by dominichamon** [comment]
sounds reasonable.

more and more i'm thinking we should move all non-json reporting to
separate tooling.

Dominic Hamon | Google
*There are no bad ideas; only good ideas that go horribly wrong.*

**Comment by Fluxie** [comment]
I take it that a small Python application that does the trick would be preferred then?

**Comment by dominichamon** [comment]
yes. something that can take the json output and render it appropriately would be best, i think.
